src/utils/hooks/api/loans/use-get-account-positions.tsx

```diff
@@ -59,31 +68,74 @@ const getAccountPositionsStats = (
 
   const borrowAmountUSD = getPositionsSumOfFieldsInUSD('amount', borrowPositions, prices);
 
-  const collateralLendPositions = lendPositions
-    .filter(({ isCollateral }) => isCollateral)
-    .map(({ amount, currency, ...rest }) => ({
-      // MEMO: compute total value based on collateral threshold (not full lend amount value)
-      amount: amount.mul(assets[currency.ticker].collateralThreshold),
-      currency,
-      ...rest
-    }));
+  const collateralLendPositions = lendPositions.filter(({ isCollateral }) => isCollateral);
+
+  const collaterAmountPositions = collateralLendPositions.map(({ amount, currency, ...rest }) => ({
```

Comment:
Need to improve name, what about `collateralLendPositionsWithAppliedThreshold` - quite long, but explicit?